sources/engine/Stride.Input/SDL/KeyboardSDL.cs

```diff
@@ -232,7 +232,7 @@ private static class SDLKeys
                 map[KeyCode.KMinus] = Keys.OemMinus;
                 map[KeyCode.KKPMinus] = Keys.Subtract;
                 map[KeyCode.KDecimalseparator] = Keys.Decimal;
-                map[KeyCode.KKPDecimal] = Keys.Decimal;
+                map[KeyCode.KKPDecimal] = Keys.NumPadDecimal;
```

Comment:
Thx for this one :) @Eideren

Reply:
Actually I looked around a bit and it's a bit more complicated than that, look here
http://kbdlayout.info/KBDBE/ <- Period numpad
http://kbdlayout.info/kbdbene <- Comma numpad
They should both map to ``NumPadDecimal`` then

Reply:
Agree 👍